playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -152,7 +152,7 @@ private void initializePlayer() {
         player = ExoPlayerFactory.newSimpleInstance(rendererFactory, trackSelector);
         window = new Timeline.Window();
         setPlayerListeners();
-        exoPlayerView.setPlayer(player);
+        exoPlayerView.setPlayer(player, isSurfaceSecured);
```

Comment:
It seems that we potentially set the surface twice on first init - 
When we call `load` if player doesn't exist we call `initializePlayer` and it causes us to set surface in the wrapper class , and then again in `maybeChangePlayerRenderView`.
So first time calling load will call it twice?
Also, in subsequent calls to play we set surface each time again - what is the logic here? I would expect an application to either use this or that and if so it can reduce the amount of logic needed?

Reply:
done